Packs/FeedTAXII/ReleaseNotes/1_1_20.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### TAXII 2 Feed
+- Improved implementation for dealing with empty response.
```

Comment:
```suggestion
- Fixed an issue where ***fetch-indicators*** failed in some cases when there were not any indicators.
```

Reply:
It is not only for fetch-indicators, can also happen in commands that use it (`get-indicators`).

Reply:
then write:

```
Fixed an issue where the ***fetch-indicators*** and ***taxii2-get-indicators*** failed in some cases when there were not any indicators.
```

Reply:
I wrote `Fixed an issue where getting indicators failed to parse empty response.` because it happens only for one implementation of returning indicators (returning empty response `{}` and not `{"objects":[]}` as it should).
Do you think it should be changed or should include commands names?

Reply:
in my opinion yes, but its not a must. Ill let you decide.
Most of the times we change code in integrations related to specific commands.

Reply:
I'll keep it this way, because this is an API module and the change is not command-specific.